projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -1017,11 +1017,9 @@ private function get_media_item_v1_1( $media_id ) {
 					}
 				}
 
-				$response['videopress_guid']            = $info->guid;
+				$response['videopress_guid']            = $info->guid ?? null;
 				$response['videopress_processing_done'] = true;
-				if ( '0000-00-00 00:00:00' === $info->finish_date_gmt ) {
-					$response['videopress_processing_done'] = false;
-				}
+				$response['videopress_processing_done'] = isset( $info->finish_date_gmt ) && '0000-00-00 00:00:00' !== $info->finish_date_gmt ? $info->finish_date_gmt : false;
```

Comment:
With this one, would we also need to check if `$info` is an object initially, for the same reason as the above comment?

So `$response['videopress_processing_done'] = is_object($info) && isset($info->finish_date_gmt) && '0000-00-00 00:00:00' !== $info->finish_date_gmt ? $info->finish_date_gmt : false;` perhaps?